generator/src/test/java/com/scottlogic/deg/generator/cucumber/InSet.feature

```diff
@@ -3,988 +3,1651 @@ Feature: User can specify that a field value belongs to a set of predetermined o
 Background:
      Given the generation strategy is full
 
+### inSet alone ###
 Scenario: Running an 'inSet' request that includes strings with roman alphabet lowercase chars (a-z) only should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "aaa" |
-       | "aab" |
-     Then the following data should be generated:
-       |  foo  |
-       | null  |
-       | "aaa" |
-       | "aab" |
+  Given there is a field foo
+    And foo is in set:
+      | "aaa" |
+      | "aab" |
+  Then the following data should be generated:
+      |  foo  |
+      | null  |
+      | "aaa" |
+      | "aab" |
 
 Scenario: Running an 'inSet' request that includes strings with roman alphabet uppercase chars (A-Z) only should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "CCC" |
-       | "DDD" |
-     Then the following data should be generated:
-       | foo   |
-       | null  |
-       | "CCC" |
-       | "DDD" |
+  Given there is a field foo
+    And foo is in set:
+      | "CCC" |
+      | "DDD" |
+  Then the following data should be generated:
+      | foo   |
+      | null  |
+      | "CCC" |
+      | "DDD" |
 
 Scenario: Running an 'inSet' request that includes strings with roman numeric chars (0-9) only should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "012" |
-       | "345" |
-     Then the following data should be generated:
-       | foo   |
-       | null  |
-       | "012" |
-       | "345" |
-
-Scenario: Running an 'inSet' request that includes strings with both roman alphabet lowercase (a-z) and uppercase (A-Z) should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "aAbB" |
-       | "AaBb" |
-     Then the following data should be generated:
-       | foo    |
-       | null   |
-       | "aAbB" |
-       | "AaBb" |
-
-Scenario: Running an 'inSet' request that includes strings with both roman alphabet (a-z, A-Z)and numeric chars (0-9) should be successful
-     Given there is a field foo
-       And foo is in set:
-       | "Testing01" |
-       | "Testing02" |
-     Then the following data should be generated:
-       |  foo        |
-       | null        |
-       | "Testing01" |
-       | "Testing02" |
+  Given there is a field foo
+    And foo is in set:
+      | "012" |
+      | "345" |
+  Then the following data should be generated:
+      | foo   |
+      | null  |
+      | "012" |
+      | "345" |
```

Comment:
these in set tests with different roman numerals could be combined.

not important though